### DISCUSSION TITLE
Service for installations?

### DISCUSSION THREAD

**Discussion by mimosel** [leihs/leihs#1561] [Q&A / Community Support] [answered]
Hi,

after having looked at the documentation about the installation of leihs, I came to the conclusion that it would be nice to have some kind of company which installs the software for us. While browsing I found: https://github.com/leihs/leihs_documentation/tree/6b520b8b2053435d8d3c97f256c2f637d4f395d8/customer-management 
Does anyone know whether this "Offerte" is still valid or can recommend a company which is able to help us install leihs?

**Comment by neoheat** [comment, marked as answer]
Dear @mimosel 
ZHdK does not provide installation support for other organizations anymore. Please direkt your request at our partner https://functional.swiss.
